Assets/Scripts/StandardSamples/UI/Login/UILoginMenu.cs

```diff
@@ -1185,6 +1185,12 @@ private void ConnectGoogleId()
 
             signInWithGoogleManager.GetGoogleIdToken((string token, string username) => 
             {
+                if (token == null)
```

Comment:
Can this check String.IsNullOrEmpty instead of direct null comparison?

Reply:
Done!